<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WareHouseMembers</class>
 <widget class="QMainWindow" name="WareHouseMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WareHouseMembers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txtFileEntry_lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Text file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="txtFileEntry_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Text File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OUTPUT:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>102</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Basic_radioButton">
       <property name="text">
        <string>Basic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Preferred_radioButton">
       <property name="text">
        <string>Preferred</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="All_radioButton">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="name_lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterName_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="name_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterID_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterID_lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>#ID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="items_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterItem_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterItem_lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prices_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rebate_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rebate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sored_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sorted</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="date_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterDate_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Add_lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Enter_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
